data/pizza.json

```diff
@@ -1192,7 +1180,7 @@
 			"hidden_ingredients": [
 
 			],
-			"price": 89
+			"price": 90
 		},
 		"africana": {
```

Comment:
[It just says nötter now, but jordnötter is probably correct 🤷](https://sannegarden.com/meny/johanneberg/#:~:text=Ananas%2C%20Curry%2C%20Kyckling%2C%20Banan%2C%20N%C3%B6tter)